docs/conf.py

```diff
@@ -33,7 +36,9 @@
 
 master_doc = 'index'
 project = u'Read the Docs'
-copyright = u'2010-2017, Read the Docs, Inc & contributors'
+copyright = '2010-{}, Read the Docs, Inc & contributors'.format(
+    datetime.now().year
```

Comment:
Instead of using python `datetime`, use django `timezone`. It have timezone aware time.

Reply:
I think we don't depends on Django for building the RTD documentation. So, you will probably need another way to achieve this.

Reply:
@humitos  for the rtd docs `django` is imported, for the generated `conf.py` for empty projects it's not, and probably it's ok using plain datetime there, since is just a year and only used for projects without a `conf.py` file.